python/nammu/test/test_yaml_update.py

```diff
@@ -0,0 +1,37 @@
+'''
+Copyright 2015, 2016 University College London.
+
+This file is part of Nammu.
+
+Nammu is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Nammu is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Nammu.  If not, see <http://www.gnu.org/licenses/>.
+'''
+
+import pytest
+import os
+# from ..utils import update_yaml_config
+
+
+@pytest.mark.skip(reason=("pyyaml is not working in the tests yet"))
```

Comment:
On that test the error has came back:

PyYAML seems to be successfully installed:

```
Adding PyYAML 3.12 to easy-install.pth file

Installed /Users/raquelalegre/workspace/ORACC/nammu/target/jython-plugins-tmp/Lib/site-packages/PyYAML-3.12-py2.7.egg
```

But then during the tests:

```
>   import yaml
E   ImportError: No module named yaml
```

This should hopefully be fixed when we update the maven plugin that installs packages.